### DISCUSSION TITLE
[FR]: The ability to alias columns

### DISCUSSION THREAD

**Discussion by spiritualgeek** [RafaelGB/obsidian-db-folder#467] [Q&A]
### Present your request

I would like the ability to alias columns. I.E. 

`firstname: Joe` should become _"First Name"_ when working with the table

If this already exists, can you point me the right direction?

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
Currently the alias is associated to the `label` parameter of the column. You can modify it manually inside the markdown ddbb note. The edition of the column also supports spaces. So if you modify the name of the column with `First Name`, the key of the column will be renamed with `first_name` and your label will be `First Name`

**Comment by RafaelGB** [comment]
moved to discussions
